src/store/FilmReducer.js

```diff
@@ -0,0 +1,19 @@
+const initialSate = {
+  filmId: 1,
+};
+
+const CHANGE_ID = "CHANGE_ID";
```

Comment:
it would be better move out the file. Imagine you have reducer for 100 handlers. It's gonna be crazy file

Reply:
Initial state and Constants rather have in separate files but maybe in the same place